Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyScreen.kt

```diff
@@ -66,7 +68,12 @@ fun SurveyQuestionsScreen(
         questions.questionsState[questions.currentQuestionIndex]
     }
 
-    Surface(modifier = Modifier.fillMaxSize()) {
+    Surface(
+        modifier = Modifier
+            .wrapContentSize()
+            .widthIn(max = 840.dp)
```

Comment:
Ditto. 

What's the difference between the above

```
                    .fillMaxWidth()
                    .wrapContentWidth(align = Alignment.CenterHorizontally)
                    .widthIn(max = 840.dp),
```

and this?

```
            .wrapContentSize()
            .widthIn(max = 840.dp)
            .fillMaxSize()
```

Should we use just one?